RELEASE-NOTES

```diff
@@ -1,62 +1,57 @@
-libssh2 1.10
+libssh2 1.11
 
 This release includes the following enhancements and bugfixes:
 
- o adds agent forwarding support
- o adds OpenSSH Agent support on Windows
- o adds ECDSA key support using the Mbed TLS backend
- o adds ECDSA cert authentication
- o adds diffie-hellman-group14-sha256, diffie-hellman-group16-sha512,
-   diffie-hellman-group18-sha512 key exchanges
- o adds support for PKIX key reading when using ed25519 with OpenSSL
- o adds support for EWOULDBLOCK on VMS systems
- o adds support for building with OpenSSL 3
- o adds support for using FIPS mode in OpenSSL
- o adds debug symbols when building with MSVC
- o adds support for building on the 3DS
- o adds unicode build support on Windows
- o restores os400 building
- o increases min, max and opt Diffie Hellman group values
- o improves portiablity of the make file
- o improves timeout behavior with 2FA keyboard auth
- o various improvements to the Wincng backend
- o fixes reading partial packet replies when using an agent
- o fixes Diffie Hellman key exchange on Windows 1903+ builds
- o fixes building tests with older versions of OpenSSL
- o fixes possible multiple definition warnings
- o fixes potential cast issues _libssh2_ecdsa_key_get_curve_type()
- o fixes potential use after free if libssh2_init() is called twice
- o improved linking when using Mbed TLS
- o fixes call to libssh2_crypto_exit() if crypto hasn't been initialized
- o fixes crash when loading public keys with no id
- o fixes possible out of bounds read when exchanging keys
- o fixes possible out of bounds read when reading packets
- o fixes possible out of bounds read when opening an X11 connection
- o fixes possible out of bounds read when ecdh host keys
- o fixes possible hang when trying to read a disconnected socket
- o fixes a crash when using the delayed compression option
- o fixes read error with large known host entries
- o fixes various warnings
- o fixes various small memory leaks
- o improved error handling, various detailed errors will now be reported
- o builds are now using OSS-Fuzz
- o builds now use autoreconf instead of a custom build script
- o cmake now respects install directory
- o improved CI backend
- o updated HACKING-CRYPTO documentation
- o use markdown file extensions
- o improved unit tests
+ o Adds support for encrypt-then-mac (etm) MACs
+ o Adds support for AES-GCM crypto protocols
+ o Adds mbedTLS 3.x support
+ o Adds support for sk-ecdsa-sha2-nistp256 and sk-ssh-ed25519 keys
+ o Adds support for RSA certificate authentication
+ o Adds FIDO support with *_sk() functions
+ o Adds RSA-SHA2 key upgrading to OpenSSL, WinCNG, mbedTLS, OS400 backends
+ o Adds Agent Forwarding and libssh2_agent_sign()
+ o Adds support for Channel Signal message libssh2_channel_signal_ex()
+ o Adds support to get the user auth banner message libssh2_userauth_banner()
+ o Adds LIBSSH2_NO_AES_CBC and LIBSSH2_NO_DSA options
```

Comment:
More were added in several steps, this is the final list: LIBSSH2_NO_MD5, LIBSSH2_NO_HMAC_RIPEMD, LIBSSH2_NO_DSA, LIBSSH2_NO_RSA, LIBSSH2_NO_RSA_SHA1, LIBSSH2_NO_ECDSA, LIBSSH2_NO_ED25519, LIBSSH2_NO_AES_CBC, LIBSSH2_NO_AES_CTR, LIBSSH2_NO_BLOWFISH, LIBSSH2_NO_RC4, LIBSSH2_NO_CAST, LIBSSH2_NO_3DES